lib/ansible/modules/network/ios/ios_banner.py

```diff
@@ -41,7 +41,7 @@
         configured on the remote device.
```

Comment:
Please state that "in Ansible 2.4 and earlier only I(login) and I(motd) were supported."